### PR TITLE
Кулыгина -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by KulyginaJulia**

### CODE REVIEW

**Comment by KulyginaJulia** [comment]
@Anna-Mironova, @SemyonBevzuk, please review code

**Review by Anna-Mironova — COMMENTED** [review]
First argument is expected value, second argument is real value. Please fix it.

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KulyginaJuliaTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        assertEquals(7.0, z.getImaginary(), 0.0);
+    }
+
+    @Test()
+    public void checkAssignment() {
+        ComplexNumber z = new ComplexNumber(5.0, 7.0);
+        ComplexNumber z2 = z;
+
+        assertEquals(z2, z);
+    }
```
Comment (Anna-Mironova) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KulyginaJuliaTest.java]: First argument is expected value, second argument is real value. Please fix it.

**Comment by Anna-Mironova** [comment]
:+1:

**Comment by SemyonBevzuk** [comment]
:+1:

**Comment by KulyginaJulia** [comment]
@kirill-korniakov

**Pull request merged**